Articles/Blog/lowndes_fellowship_post.md

```diff
@@ -0,0 +1,33 @@
+
+## A new learning paradigm of trust and empowerment
+
+It was through Jenny Bryan that I first felt this trust and empowerment as a new coder and learner. Jenny Bryan, then a statistics professor at University of British Columbia and now a software engineer at Posit (formerly RStudio), shared her [Stat 545 Course](https://stat545.com/) materials as open source book (made originally with RMarkdown and now Quarto) that were a delight and light-hearted to read as it walks you through highly technical material for data wrangling, exploration, and analysis with R. Jenny went on to create additional books that are still my go-to resources and recommendations: [Happy Git With R](https://happygitwithr.com/) (which is in my opinion the best place to learn Git/GitHub, regardless of whether you are an R user), and [What They Forgot to Teach You About R](https://rstats.wtf/) (designed for an audience that "has a moderate amount of R and RStudio experience; is largely self-taught; suspects they have drifted into some idiosyncratic habits that may slow them down or make their work products more brittle; is interested in (re)designing their R lifestyle, to be more effective and more self-sufficient").
+
+When she teaches, Jenny provides these resources openly to her students, breaking the paradigm of the teacher having all the power and withholding information from the learners. Instead, she trusts that we we are there to learn, and having these resources in front of us reinforces that goal. And since these books are also Jenny's teaching scripts, it also has the effect of having everything she needs in one place. Being open with her script and sharing it with us in advance, we feel trusted and empowered as peers rather than infantalizing us as learners. As a learner, having the book open in my browser meant that I could reference it if I had been looking at my keyboard instead of the screen, and didn't need to interrupt the class to say "wait could you repeat that?" Sidenote: Jenny live-codes as she teaches, role-modeling that coders make typos and get errors and look things up, and further building confidence in ourselves as learners.
```

Comment:
```suggestion
When she teaches, Jenny provides these resources openly to her students, breaking the paradigm of the teacher being the sole source of information; she is teaching live what I can also reference from a tab in my browser. As a learner, having the book open in my browser helps me because looking between my own console and the teacher's screen means I can miss syntax and now I don't fall behind or have to interrupt the class to say "could you scroll up ?". Futher, since these books are also Jenny's teaching narrative, it also has the effect of having everything she needs in one place. Being open with her script and sharing it with us in advance, we feel trusted and empowered as peers. Sidenote: Jenny live-codes as she teaches, role-modeling that coders make typos and get errors and look things up, and further building confidence in ourselves as learners.
```